torch/_inductor/graph.py

```diff
@@ -248,8 +248,9 @@ def placeholder(self, target: str, args, kwargs):
         example: torch.Tensor = super().placeholder(target, args, kwargs)
         if config.static_weight_shapes and (
             len(self.graph_inputs) < self.num_static_inputs or not config.dynamic_shapes
-        ):
+        ) and not example._has_symbolic_sizes_strides:
             # the first N inputs are weights
+            # print(len(self.graph_inputs), self.num_static_inputs)
```

Comment:
per Horace, this is a hack, the static tensors shouldn't have been passed in in fakeified form